app/models/gifts/gifts-model.test.ts

```diff
@@ -0,0 +1,134 @@
+import { allSettled, fork } from 'effector';
+import { beforeEach, describe, expect, test, vi } from 'vitest';
+
+import { BN_ZERO } from '@polkadot/util';
+
+import { networkModel } from '@/models/network';
+import { TelegramApi, balancesFactory, localStorageApi } from '@/shared/api';
+import { type Gift } from '@/types/substrate';
+
+import { giftsModel } from './gifts-model';
+
+describe('models/gifts/gifts-model', () => {
+  beforeEach(() => {
+    vi.resetAllMocks();
+  });
+
+  test('should request local gifts on giftsRequested', async () => {
+    vi.spyOn(TelegramApi, 'getStoreName').mockReturnValue('store');
+    vi.spyOn(localStorageApi, 'getItem').mockReturnValue([
```

Comment:
do you really need to mock localStorage and service to run unit test? I think it's better to replace effect handler in using scope